code/game/gamemodes/miniantags/guardian/types/protector.dm

```diff
@@ -121,7 +125,7 @@
 	shield_orientation = left_or_right
 
 /obj/effect/guardianshield/CanPass(atom/movable/mover, turf/target)
-	if(mover == linked_guardian || mover == linked_guardian.summoner)
+	if(mover == linked_guardian)
```

Comment:
can you edit the `playstyle_string` to reflect this change?

Reply:
I don't see anything that needs editing there

Reply:
Something just to tell people that their summoner can't pass through their shield, because it has a lot of chances to fuck up people who don't know.

Reply:
Sure